lib/doorkeeper/oauth/token.rb

```diff
@@ -26,7 +26,7 @@ def from_basic_authorization(request)
 
         def token_from_basic_header(header, pattern)
           encoded_header = token_from_header(header, pattern)
-          token, _ = decode_basic_credentials(encoded_header)
+          token, = decode_basic_credentials(encoded_header)
```

Comment:
This is such weird syntax. How can we rewrite this in a clearer way than either the older or the newer option?